packages/api/src/hooks/useJurisdictionStatus.ts

```diff
@@ -35,8 +35,8 @@ const useJurisdictionStatus = () => {
                 }
             };
 
-            const pendingStatuses = ['poa_pending', 'verification_pending'];
```

Comment:
`poa_pending` is removed from the pending status check as client can already start trading with their MT5 account and click on the button once POI is verified